language/stdlib/modules/AccountFreezing.move

```diff
@@ -105,11 +105,15 @@ module AccountFreezing {
     /// Returns if the account at `addr` is frozen.
     public fun account_is_frozen(addr: address): bool
     acquires FreezingBit {
-        borrow_global<FreezingBit>(addr).is_frozen
+        exists<FreezingBit>(addr) && borrow_global<FreezingBit>(addr).is_frozen
      }
 
```

Comment:
I believe you need to add 
```
spec fun account_is_frozen {
   pragma verify = true;
   ensures spec_account_is_frozen(addr);
}
```
to verify the spec fun against the impl (is that right and/or is there a more concise way @junkil-park)?

Reply:
Yes, that's right. `pragma verify = true;` can be omitted because verification is already turned on at the module level (i.e., "pragma verify" in "spec module"). Also, "aborts_if false;" could be added.